ports/stm32f4/boards/stm32f412zg_discovery/stm32f4xx_hal_msp.c

```diff
@@ -273,14 +273,14 @@ void HAL_I2C_MspInit(I2C_HandleTypeDef* hi2c)
     */
     GPIO_InitStruct.Pin = I2C2_SCL_Pin;
```

Comment:
Are you going to move these pin inits to I2C.c?

Reply:
My plan is to entirely revamp the pin and init system to remove the MSC files and have everything done via the LL libraries, yes.